packages/mgt/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -5,184 +5,187 @@
-     .expanded-details-button {
-       display: flex;
-       justify-content: center;
-       align-items: center;
-       height: 28px;
-       background: rgba(196, 196, 196, 0.1);
-       .expanded-details-svg {
-         margin-bottom: 3px;
-       }
-     }
-     .expanded-details-button:hover {
-       background-color: rgba(0, 0, 0, 0.068);
-     }
-     .expanded-details-info {
-       margin: 0 0 0 20px;
-       padding-bottom: 18px;
-       display: flex;
-       flex-direction: column;
- 
-       .contact-text {
-         margin: 0 0 24px 0;
-         font-weight: 600;
-         font-size: $person-card-details-title-font-size;
-         color: $person-card-details-title-color;
-       }
- 
-       .icons {
-         line-height: normal;
-         font-style: normal;
-         font-weight: normal;
-         font-size: $person-card-details-item-font-size;
-         color: $person-card-details-item-color;
-         text-align: left;
- 
-         .details-icon {
-           margin: 0 0 10px 0;
-           display: flex;
-           flex-direction: row;
-           align-items: center;
-         }
-       }
-       .data {
-         display: block;
-         margin: 0 0 0px 10px;
-         white-space: nowrap;
-         overflow: hidden;
-         text-overflow: ellipsis;
-         width: 280px;
-       }
-       .normal-subtitle {
-         padding-top: 2px;
-       }
-       .link-subtitle {
-         color: #3078cd;
-         cursor: pointer;
-       }
-       .link-subtitle:hover {
-         opacity: 0.6;
-       }
-     }
-     .section-divider {
-       width: 96%;
-       height: 1px;
-       background: #eaeaea;
-     }
-   }
- }
\ No newline at end of file
+@import '../../styles/shared-styles.scss';
+@import '../../../node_modules/office-ui-fabric-core/dist/sass/References';
+
+$person-card-display-name-font-size: var(--person-card-display-name-font-size, 27.3103px);
+$person-card-title-font-size: var(--person-card-title-font-size, 18.2069px);
+$person-card-subtitle-font-size: var(--person-card-subtitle-font-size, 15.1724px);
+$person-card-details-title-font-size: var(--person-card-details-title-font-size, 14px);
+$person-card-details-item-font-size: var(--person-card-details-item-font-size, 15px);
+$person-card-display-name-color: var(--person-card-display-name-color, #333333);
+$person-card-title-color: var(--person-card-title-color, #767676);
+$person-card-subtitle-color: var(--person-card-subtitle-color, #767676);
+$person-card-details-title-color: var(--person-card-details-title-color, #000000);
+$person-card-details-item-color: var(--person-card-details-item-color, #201f1e);
+$person-card-background-color: var(--person-card-background-color, #ffffff);
+
+:host .root {
+  position: relative;
+  background: $person-card-background-color;
+  width: var(--mgt-flyout-set-width, 340px);
+  font-family: var(--default-font-family);
+
+  .person-details-container {
+    display: flex;
+    padding: 18px 14px;
+
+    mgt-person {
```

Comment:
Can you please replace the mgt-person with an real classname. This helps for another thing I'm working on.

Reply:
I have addressed this :)